contributors/devel/issues.md

```diff
@@ -12,33 +80,62 @@ We try to apply these priority labels consistently across the entire project,
+someone is expected to drop what they're doing immediately to work on it. Team
+leaders are responsible for making sure that all P0's in their area are being
+actively worked on. Examples include user-visible bugs in core features, broken
+builds or tests and critical security issues.
 
-- **priority/failing-test**: Automatically filed frequently failing test. Needs to be investigated.
+- **priority/failing-test**: Automatically filed frequently failing test. Needs
+to be investigated.
 
-- **priority/important-soon**: Must be staffed and worked on either currently, or very soon,
-ideally in time for the next release.
+- **priority/important-soon**: Must be staffed and worked on either currently,
+or very soon, ideally in time for the next release.
 
-- **priority/important-longterm**: Important over the long term, but may not be currently
-staffed and/or may require multiple releases to complete.
+- **priority/important-longterm**: Important over the long term, but may not be
+currently staffed and/or may require multiple releases to complete.
 
-- **priority/backlog**: There appears to be general agreement that this would be good
-to have, but we may not have anyone available to work on it right now or in the
-immediate future. Community contributions would be most welcome in the mean time
-(although it might take a while to get them reviewed if reviewers are fully
+- **priority/backlog**: There appears to be general agreement that this would be
+good to have, but we may not have anyone available to work on it right now or in
+the immediate future. Community contributions would be most welcome in the mean
+time (although it might take a while to get them reviewed if reviewers are fully
 occupied with higher priority issues, for example immediately before a release).
 
-- **priority/awaiting-more-evidence**: Possibly useful, but not yet enough support to actually get
-it done. These are mostly place-holders for potentially good ideas, so that they
-don't get completely forgotten, and can be referenced/deduped every time they
-come up.
+- **priority/awaiting-more-evidence**: Possibly useful, but not yet enough
+support to actually get it done. These are mostly place-holders for potentially
+good ideas, so that they don't get completely forgotten, and can be referenced
+/deduped every time they come up.
+
+## Set ownership
+
+If you are not sure of who should own issue, defer to the
+SIG label only. If you feel the issue should warrant a notification,you can ping
+a team with an @ mention, in this format, `@kubernetes/sig-<group-name>-<group-suffix>`.
+Here the `<group-suffix>` can be one of `bugs, feature-requests,pr-reviews, test-failures, proposals`.
+For example, `@kubernetes/sig-cluster-lifecycle-bugs, can you have a look at this?`
+
+If you think you can fix the issue and you are an issue reporter or a component
+organization member, assign it to yourself with just `/assign`. If you can not
```

Comment:
FYI, I created k8s-external-contributor as a placeholder for issues being worked on by non-members.